frontend/server/fileSystem.js

```diff
@@ -4,6 +4,42 @@ import fs from "fs/promises";
 import { tmpdir } from "os";
 import path from "path";
 
+//fix for specs.json issue
+const coreBlockDir = 'core/blocks'; 
+
+// Function to copy specs_v1.json to specs.json if specs.json doesn't exist
+export const copySpecsIfNotExists = async (dir) => {
+  const specsV1Path = path.join(dir, 'specs_v1.json');
+  const specsPath = path.join(dir, 'specs.json');
+  
+  if (await fileExists(specsV1Path) && !(await fileExists(specsPath))) {
+    const specsV1Data = await fs.readFile(specsV1Path, 'utf8');
+    await fs.writeFile(specsPath, specsV1Data);
+    console.log(`Replicated contents in ${specsV1Path} to ${specsPath}`);
+  } else {
+    console.log(`specs.json already exists in ${dir}`); //logs if specs.json already exists or specs_v1.json does not exist
+  }
+}
+
+// Function to process all processing blocks in the core block directory
+const processCoreBlockDirectory = async (coreBlockDir) => {
+  try {
+    const processingBlocks = await fs.readdir(coreBlockDir);
+    for (const block of processingBlocks) {
+      const blockPath = path.join(coreBlockDir, block);
+      const stat = await fs.stat(blockPath);
+      if (stat.isDirectory()) {
+        await copySpecsIfNotExists(blockPath);
+      }
+    }
+  } catch (error) {
+    console.error("Error processing core block directory:", error);
+  }
+}
+
+//call function to process core block directory
+processCoreBlockDirectory(coreBlockDir);
```

Comment:
Instead of doing this here I think we should check during the load process for this name and if it's `specs_v1.json` we should modify the filename and save it as `specs.json`, that way we would handle older block versions that may not be part of the core/blocks library

Reply:
Ok, I'll work on that, I did it this way,  because this is similar to how it is handled using the load block button from the dropdown menu.

Reply:
I agree. Instead of converting the spec files when the app start, we can just try to load both `specs_v1.json` and `specs.json` when something tries to access the specs for block in the `readSpecs` function.